troubadix/plugins/newlines.py

```diff
@@ -36,20 +36,25 @@ def run(
         - Search for (\r or \r\n).
         - Search for whitespaces in script_name( "myname") or script_copyright
         """
-
-        if "\r" in file_content:
-            yield LinterError("Found \r.")
+        # Need to be loaded as bytes or \r is converted to \n
+        data = nasl_file.read_bytes().decode(CURRENT_ENCODING)
```

Comment:
I really really thing this is not what you want. The code is now the same as `nasl_files.read_text(encoding=CURRENT_ENCODING)`

Reply:
if i use this i dont find the /r in fail_bad_new_line.nasl.
Or my unittest for it is broken

Reply:
ok changed it to b"/r" in bytes.